gradle/libs.versions.toml

```diff
@@ -163,6 +164,8 @@ scala-collection-compat = { module = "org.scala-lang.modules:scala-collection-co
 slf4j-api = { module = "org.slf4j:slf4j-api", version.ref = "slf4j" }
 slf4j-simple = { module = "org.slf4j:slf4j-simple", version.ref = "slf4j" }
 snowflake-jdbc = { module = "net.snowflake:snowflake-jdbc", version.ref = "snowflake-jdbc" }
+vortex-jni = { module = "dev.vortex:vortex-jni", version.ref = "vortex" }
+vortex-spark = { module = "dev.vortex:vortex-spark", version.ref = "vortex" }
```

Comment:
I have not setup maven central publishing for these jars so just hosting them out of mavenLocal for the moment